### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by DefaultDaddy**
Added PlayWright test to application. This means we have created UI tests, which works. We still have issues with Playwright and accessing database, which will be addressed in another pull request.

### CODE REVIEW

**Comment by DefaultDaddy** [comment]
The above merge conflicts will be solves by accepting the main version of the files, since Bjørn has also fixed the addressed issues in these files

File: PlaywrightTests/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+namespace PlaywrightTests;
```
Comment (alexop1000) [inline PlaywrightTests/UnitTest1.cs:1]: unused code?

File: playwrighttest/example.spec.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// @ts-check
```
Comment (alexop1000) [inline playwrighttest/example.spec.js:1]: should remove this file too

File: src/Chirp.Razor/Program.cs [hunk 3]
```diff
@@ -76,7 +76,10 @@
     // From the scope, get an instance of our database context.
     // Through the using keyword, we make sure to dispose it after we are done.
     using var context = scope.ServiceProvider.GetService<ChirpDBContext>();
-
+    
+    // Wiping the database
+    DbInitializer.WipeDatabase(context);
```
Comment (alexop1000) [inline src/Chirp.Razor/Program.cs:81]: It shouldn't wipe the database.

File: test/Chirp.CSVDBService/UnitTest1.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
+using Chirp.Core.Classes;
```
Comment (alexop1000) [inline test/Chirp.CSVDBService/UnitTest1.cs:1]: Should this file be here? We don't have a CSV database anymore

File: src/Chirp.Razor/Program.cs (hunk 3 above)
Comment (DefaultDaddy) [reply src/Chirp.Razor/Program.cs:81]: Yes it should!

**Review by MrBStones — CHANGES_REQUESTED** [review]
Same as alex.

Also two tests are still failing:
```
[xUnit.net 00:00:01.76]     Chirp.Razor.Tests.IntegrationTests.CanSeePrivateTimeline(author: "Jacqualine Gilcoine", authorId: 10) [FAIL]
  Failed Chirp.Razor.Tests.IntegrationTests.CanSeePrivateTimeline(author: "Jacqualine Gilcoine", authorId: 10) [154 ms]
  Error Message:
   Assert.Contains() Failure: Sub-string not found
String:    "\n<!DOCTYPE html>\n<html lang="en">\n<script"···
Not found: "Jacqualine Gilcoine's Timeline"
  Stack Trace:
     at Chirp.Razor.Tests.IntegrationTests.CanSeePrivateTimeline(String author, Int32 authorId) in /home/mrbstones/ITUGIT/Chirp/test/Chirp.Razor.Tests/ChirpTests.cs:line 507
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
[xUnit.net 00:00:01.92]     Chirp.Razor.Tests.IntegrationTests.CanSeePrivateTimeline(author: "Quintin Sitts", authorId: 5) [FAIL]
  Failed Chirp.Razor.Tests.IntegrationTests.CanSeePrivateTimeline(author: "Quintin Sitts", authorId: 5) [161 ms]
  Error Message:
   Assert.Contains() Failure: Sub-string not found
String:    "\n<!DOCTYPE html>\n<html lang="en">\n<script"···
Not found: "Quintin Sitts's Timeline"
  Stack Trace:
     at Chirp.Razor.Tests.IntegrationTests.CanSeePrivateTimeline(String author, Int32 authorId) in /home/mrbstones/ITUGIT/Chirp/test/Chirp.Razor.Tests/ChirpTests.cs:line 507
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)

Failed!  - Failed:     2, Passed:    20, Skipped:     0, Total:    22, Duration: 1 s - Chirp.Razor.Tests.dll (net8.0)
```

**Pull request merged**